React/src/screens/Dashboard/screens/Home/index.tsx

```diff
@@ -1,70 +1,47 @@
+import LogoWolox from '../../../../resources/img/logoWolox.png';
 
-interface TechForm {
-  tech: string;
-}
+import styles from './styles.module.scss';
+import { withContextProvider } from './context';
 
 function Home() {
-  const { t, i18n } = useTranslation();
-
-  // Example of how to use these custom hooks
-  const tech = useSelector((state) => state.tech);
-  const dispatch = useDispatch();
-  const userDispatch = useUserDispatch();
-  const { register, handleSubmit } = useForm<TechForm>();
-
-  const handleLogout = async () => {
-    await logout();
-    userDispatch(authActions.resetUser());
-    removeCurrentUserToken();
-  };
-
-  const handleChangeLanguage = () => {
-    i18n.changeLanguage(i18n.language === 'es' ? 'en' : 'es');
-  };
-
-  const onSubmit = handleSubmit((values) => {
-    if (values.tech) {
-      dispatch(actionCreators.setTech(values.tech));
-    }
-  });
+  const { register, handleSubmit } = useForm<UserData>();
 
   return (
-    <div className={styles.app}>
-      <header className={styles.appHeader}>
-        <img src={logo} className={styles.appLogo} alt="logo" />
-        <p className={styles.text}>{t('Home:loggedIn')}</p>
-        <p className={styles.text}>{t('Home:techIs', { tech })}</p>
-        <form className="column center m-bottom-10" onSubmit={onSubmit}>
-          <FormInput
-            className="m-bottom-2"
-            placeholder={t('Home:newTech')}
-            inputRef={register('tech')}
-            name="tech"
-            inputType="text"
-          />
-          <button className={styles.appLink} type="submit">
-            {t('Home:setNewTech')}
+    <div className={styles.pageContainer}>
+      <div className={styles.loginContainer}>
+        <img src={LogoWolox} alt="Wolox Logo Image" />
+        <form
+          onSubmit={handleSubmit((data) => console.log({ users: data }))}
+          className={styles.inputsContainer}
+        >
+          <h2 className={styles.dataTitle}>Nombre</h2>
+          <input {...register('name')} className={styles.dataInput} type="text" />
+
+          <h2 className={styles.dataTitle}>Apellido</h2>
+          <input {...register('lastName')} className={styles.dataInput} type="text" />
+
+          <h2 className={styles.dataTitle}>Email</h2>
+          <input {...register('email')} className={styles.dataInput} type="text" />
+
+          <h2 className={styles.dataTitle}>Password</h2>
+          <input {...register('password')} className={styles.dataInput} type="password" />
+
+          <h2 className={styles.dataTitle}>Confirmación De Password</h2>
+          <input {...register('passwordConfirmation')} className={styles.dataInput} type="password" />
+
+          <button type="submit" className={styles.loginButton}>
+            <p className={styles.loginTitle}>Login</p>
+          </button>
+          <div className={styles.line} />
+          <button type="submit" className={styles.signUpButton}>
+            <p className={styles.signUpTitle}>Sign up</p>
```

Comment:
You should work with clean screen components, all this logic and layout should be placed on a different component, and then call the component from this screen.

Moreover, you must analize the current layout code and check if you are repeating code that meaybe could me place in a separated component.